stdlib/public/core/Integers.swift

```diff
@@ -3720,24 +3581,10 @@ extension SignedInteger {
     fatalError("Should be overridden in a more specific type")
   }
 
-  @_transparent
-  @available(swift, obsoleted: 4.0,
-      message: "Please use 'FixedWidthInteger' instead of 'SignedInteger' to get '&+' in generic code.")
-  public static func &+ (lhs: Self, rhs: Self) -> Self {
-    return _maskingAdd(lhs, rhs)
-  }
-
   @_transparent
   public static func _maskingSubtract(_ lhs: Self, _ rhs: Self) -> Self {
     fatalError("Should be overridden in a more specific type")
   }
-
-  @_transparent
-  @available(swift, obsoleted: 4.0,
-      message: "Please use 'FixedWidthInteger' instead of 'SignedInteger' to get '&-' in generic code.")
-  public static func &- (lhs: Self, rhs: Self) -> Self {
-    return _maskingSubtract(lhs, rhs)
-  }
```

Comment:
Some of these things are here to help when people type the wrong thing. Are you sure it's worth removing them?

Reply:
Yeah, that was when `SignedInteger` implied a fixed-width-integer-ness, which has not been the case for more than a year no, so I'd say, yes.